website/common/locales/en/faq.json

```diff
@@ -1,133 +1,136 @@
+  "general": "Common Questions",
+
+  "faqQuestion0": "What are the different task types?",
+  "webFaqAnswer0": "Habitica uses three different task types to accommodate your needs: Habits, Dailies, and To Do’s.\n\nHabits can be positive or negative and represent something you may want to track multiple times per day, or on an unset schedule. Positive Habits will provide you with rewards, like Gold and Experience (Exp), while Negative Habits will cause you to lose health points (HP).\n\nDailies are repeated tasks you want to complete on a more structured schedule. E.g. Once per day, three times a week, or four times a month. Missing Dailies causes you to lose HP, but the more difficult they are, the better the rewards!\n\nTo Do’s are one-off tasks that provide rewards after you complete them. To Do’s can have a due date, but you won’t lose HP if you miss it.\n\nPick the task type that best fits what you want to achieve!",
+
+  "faqQuestion1": "What are some sample tasks?",
```

Comment:
These no longer line up semantically with the prior versions of the questions. Sample tasks were question 2, now they're question 1. We can add new indexes and remove unused ones, but the whole point of the `translationIndex` was so we didn't break the meaning of translations, which this does.

Reply:
We knew that this would break the existing `translationIndex` since we were creating the starting content for the new FAQ.